conference.js

```diff
@@ -553,16 +577,21 @@ export default {
                                 && type.indexOf('video') !== -1) {
                                 return createLocalTracks(['audio'], null,
                                     micDeviceId);
+                            } else if (type.indexOf('audio') !== -1) {
+                                audioOnlyError = err;
+                            } else if (type.indexOf('video') !== -1) {
```

Comment:
In my opinion, the following does less indexOf lookups (in addition to being easier on my eyes because I don't have to check multiple times which one of the two alternatives a given one of the four indexOf is actually looking for):
a = (type.indexOf('audio') !== -1);
v = (type.indexOf('video') !== -1);
if (a && v)
else if (a)
else if (v)

Then I find the following to not rely on any supposed JS compiler optimizations (besides being easier for me to understand because it doesn't compose complex conditions)
if (a) {
  if (v) {
    audio & video
  } else {
    audio only
  }
} else if (v) {
 video only
}

What do you think?